src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/tests/test_sql_commands.py

```diff
@@ -62,3 +62,135 @@ def body(self):
 
         # test list sql server should be 0
         self.cmd('sql server list -g {}'.format(rg), checks=[JMESPathCheck('length(@)', 0)])
+
+class SqlServerFirewallMgmtScenarioTest(ResourceGroupVCRTestBase):
```

Comment:
two blank lines between class and function definition. 

[Improve your code style](https://github.com/Azure/azure-cli/wiki/Improve-your-code-style)